test/ref/round.cpp

```diff
@@ -37,7 +37,7 @@ TEST_CASE(round_test)
     migraphx::shape s{migraphx::shape::float_type, {9}};
     auto l =
         mm->add_literal(migraphx::literal{s, {1.1, 1.5, 1.6, -1.1, -1.5, -1.6, 0.0, 2.0, -2.0}});
-    mm->add_instruction(migraphx::make_op("round"), l);
+    mm->add_instruction(migraphx::make_op("nearbyint"), l);
```

Comment:
Sorry for the overlapping question elsewhere. If this test is for the round() operator, why change it by inserting a nearbyint operator, and still leave both operators in existence. One of them should likely go away, it seems, and maybe just the round() operator should be fixed, and still called round. Thanks.

Reply:
{1.1, 1.5, 1.6, -1.1, -1.5, -1.6, 0.0, 2.0, -2.0}  --> please add some more test values here to test even rounding. 

Suggestion, please add (or something similar) : -1.4,  0.5, 0.6, 2.4, 2.5, 2.6, -2.4, -2.5, -2.6.